care/facility/models/bed.py

```diff
@@ -82,6 +82,7 @@ class ConsultationBed(BaseModel):
     bed = models.ForeignKey(Bed, on_delete=models.PROTECT, null=False, blank=False)
     start_date = models.DateTimeField(null=False, blank=False)
     end_date = models.DateTimeField(null=True, blank=True, default=None)
+    privacy = models.BooleanField(default=False)
```

Comment:
We are not capturing which user has toggled the privacy and at what time. Could be a separate issue.

Reply:
Yeah, I discussed this with @khavinshankar only users like doctor and above have access that's why we aren't storing it right now